README.md

```diff
@@ -354,6 +355,26 @@ _Mailing lists servers and mass mailing software - one message to many recipient
 - [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPL-2.0` `PHP`
 - [WebMail Lite](http://www.afterlogic.org/webmail-lite) - Web-based IMAP Mail client. ([Source Code](https://github.com/afterlogic/webmail-lite)) `GPL-3.0` `PHP`
 
+#### Disposable email and forwarding services
+
+**[`^        back to top        ^`](#)**
+
+- [Ad Hoc Email Server (AHEM)](http://www.ahem.email) - An ad-hoc mail server. Create and use ad hoc disposable mailboxes for testing or any other purpose. http://www.ahem.email ([Demo](http://www.ahem.email/account/bi1vx86r), [Source Code](https://github.com/o4oren/Ad-Hoc-Email-Server)) `Angular/Nodejs` `MongoDB` `MIT`
+- [Alp-Mail](https://github.com/theju/alp-mail) - Alp-Mail (Alp in Hindi means short-lived) is your self-hosted disposable email service ala Mailinator. All mails are stored in a database under incoming or rejected. ([Source Code](https://github.com/theju/alp-mail)) `Nodejs` `Django` `MIT`
```

Comment:
Don't duplicate links. Delete the link to the source code since the primary project link is in fact the same link to the source code.

Reply:
Removed, My apologies, when I read the posting guide it said, 

>there is no need to add a duplicate Source Code link. 

I read that as optional, and since I prefer structure wanted everything to look as similar as possible. That and I was just copy/pasting

- []() -  ([Demo](), [Source Code]()) `` ``

over a few times then going back and filling in the blanks, will conform to new understanding from here on out.